charts/linkerd2/values.yaml

```diff
@@ -4,7 +4,6 @@
 
 ClusterDomain: &cluster_domain cluster.local
 EnableH2Upgrade: true
-EnablePodAntiAffinity: false
 ImagePullPolicy: &image_pull_policy IfNotPresent
 
 # control plane version. See Proxy section for proxy version
```

Comment:
unrelated to this PR, should this be `edge-19.8.2`, or something else?

Reply:
Yes, it should. We are yet to figure out a way to inject the new version numbers into these static files (`values.yaml` and `Chart.yaml`) as part of the build.